addons/crm/models/crm_lead.py

```diff
@@ -2000,15 +2002,17 @@ def _pls_get_lead_pls_values(self, domain=[]):
             self._cr.execute(query, where_params)
             lead_results = self._cr.dictfetchall()
 
-            # Get tags values
-            query = """SELECT crm_lead.id as lead_id, t.id as tag_id
-                            FROM %s
-                            LEFT JOIN crm_tag_rel rel ON crm_lead.id = rel.lead_id
-                            LEFT JOIN crm_tag t ON rel.tag_id = t.id
-                            WHERE %s order by crm_lead.team_id asc"""
-            query = sql.SQL(query % (from_clause, where_clause)).format(*args)
-            self._cr.execute(query, where_params)
-            tag_results = self._cr.dictfetchall()
+            if use_tags:
+                # Get tags values
+                query = """SELECT crm_lead.id as lead_id, t.id as tag_id
```

Comment:
Linting: could be indented a bit better (not enforcing black but seems somehow we are heading to it).